client/my-sites/domains/domain-management/list/all-domains.jsx

```diff
@@ -260,75 +233,6 @@ class AllDomains extends Component {
 		} ) );
 	}
 
-	renderQuerySiteDomainsOnce = ( blogId ) => {
-		if ( this.renderedQuerySiteDomains[ blogId ] ) {
-			return null;
-		}
-		this.renderedQuerySiteDomains[ blogId ] = true;
-		return <QuerySiteDomains siteId={ blogId } />;
-	};
-
-	getActionResult( domain ) {
-		if ( this.props.isContactEmailEditContext ) {
-			return this.state.contactInfoSaveResults[ domain ] ?? null;
-		}
-
-		return null;
-	}
-
-	renderDomainItem( domain, index ) {
```

Comment:
Not used, this functionality is now done inside `DomainsTable`'s render method, which renders `DomainRow` components